index.bs

```diff
@@ -5624,13 +5624,20 @@ The <dfn export for=commands>emulation.setGeolocationOverride</dfn> command modi
         params: emulation.SetGeolocationOverrideParameters
       )
 
-      emulation.SetGeolocationOverrideParameters = {
-        (
-          (coordinates: emulation.GeolocationCoordinates / null) //
-          (error: emulation.GeolocationPositionError)
-        ),
+      emulation.SetGeolocationOverrideParameters = (
         ? contexts: [+browsingContext.BrowsingContext],
         ? userContexts: [+browser.UserContext],
+        (emulation.SetGeolocationOverrideSuccessParameters // emulation.SetGeolocationOverrideErrorParameters)
```

Comment:
I believe SetGeolocationOverrideSuccessParameters and SetGeolocationOverrideErrorParameters have to be groups (`()`), not types (map `{}`).

Reply:
That is correct, updated!